Backbone.API/appsettings.override.json

```diff
@@ -36,6 +36,16 @@
         }
       }
     },
+    "Quotas": {
+      "Application": {},
+      "Infrastructure": {
+        "SqlDatabase": {
+          "Provider": "SqlServer",
+          //"ConnectionString": "User ID=challenges;Password=Passw0rd;Server=postgres;Port=5432;Database=enmeshed;" // postgres
+          "ConnectionString": "Server=ms-sql-server;Database=enmeshed;User Id=challenges;Password=Passw0rd;TrustServerCertificate=True"
```

Comment:
The connection strings are using the 'challenges' user instead of the quotas one.